### PR TITLE
Re-enable mpOpenTracing Rest Client Tests

### LINKED ISSUE #14222: Re-enable Open Tracing Rest Client TCK tests
When mpRestClient-2.0 is ready and supports mpConfig-2.0, the Open Tracing Rest Client TCKs should be re-enabled.

https://github.com/OpenLiberty/open-liberty/blob/1a9c568e83a80da8c34ac02f250aa7d49fa546c3/dev/io.openliberty.microprofile.opentracing.2.0.internal_fat_tck/fat/src/io/openliberty/microprofile/opentracing/internal/tck/FATSuite.java#L31

### PR DESCRIPTION
**Pull request by fmhwong**
PR #13949 temporary disabled mpOT 2.0 rest client tests.  This PR re-enable them after mpRestClient-2.0 is delivered.

Fixes #14222

### CODE REVIEW

**Comment by fmhwong** [comment]
#libby
#build

File: dev/io.openliberty.microprofile.opentracing.2.0.internal_fat_tck/fat/src/io/openliberty/microprofile/opentracing/internal/tck/FATSuite.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
     OpentracingTCKLauncher.class,
 // Temporary comment this out until MicroProfile-4.0 is available ... including MP Rest Client 2.0
 //    OpentracingTCKLauncherMicroProfile.class,
```
Comment (tevans78) [inline dev/io.openliberty.microprofile.opentracing.2.0.internal_fat_tck/fat/src/io/openliberty/microprofile/opentracing/internal/tck/FATSuite.java:32]: Should be able to re-enable this as well once https://github.com/OpenLiberty/open-liberty/pull/14560 is merged

**Review by tevans78 — APPROVED** [review]
Approved but should also re-enable OpentracingTCKLauncherMicroProfile when #14560 is merged.

**Review by andymc12 — APPROVED** [review]
LGTM - thanks @fmhwong!

**Pull request merged**